moto/wafv2/utils.py

```diff
@@ -0,0 +1,14 @@
+import uuid
+from collections import namedtuple
+
+ARN = namedtuple("ARN", ["region", "account", "function_name", "version"])
+
+
+def make_arn(service, region, name):
+    return "arn:aws:wafv2:{0}:123456789012:regional/{1}/{2}/{3}".format(
```

Comment:
The account ID is stored centrally:
`from moto.core import ACCOUNT_ID`

Reply:
Good to know - updated... thanks!